resolvers/node/resolution.rs

```diff
@@ -632,9 +668,7 @@ impl<
       &package_subpath,
       maybe_referrer.as_ref(),
       resolution_mode,
-      self
-        .conditions_from_resolution_mode
-        .resolve(resolution_mode),
+      &self.condition_resolver,
```

Comment:
Can you revert this change? It was like this way in order to support specific imports providing custom conditions. You can get the types conditions by adding a method like `self.condition_resolver.types_conditions()` and that way someone can provide conditions to these methods without needing knowledge of the current configuration.